### ISSUE TITLE
doesn't work on Java 9+ (FileNotFoundException)

### ISSUE THREAD

**Issue by SethTisue** [chipsenkbeil/sbt-jdi-tools#5]
```
% mkdir jdi
% cd jdi
% mkdir project                                                            
% echo 'addSbtPlugin("org.scala-debugger" % "sbt-jdi-tools" % "1.1.0")' > project/plugins.sbt
% echo 'sbt.version=1.1.6' > project/build.properties
% usejava9
% java -version
java version "9.0.4"
Java(TM) SE Runtime Environment (build 9.0.4+11)
Java HotSpot(TM) 64-Bit Server VM (build 9.0.4+11, mixed mode)
% sbt
[info] Loading settings from plugins.sbt ...
[info] Loading project definition from /Users/tisue/tmp/jdi/project
[info] Updating ProjectRef(uri("file:/Users/tisue/tmp/jdi/project/"), "jdi-build")...
[info] downloading http://127.0.0.1:8081/artifactory/scala-ci-virtual/org/scala-debugger/sbt-jdi-tools_2.12_1.0/1.1.0/sbt-jdi-tools-1.1.0.jar ...
[info] 	[SUCCESSFUL ] org.scala-debugger#sbt-jdi-tools;1.1.0!sbt-jdi-tools.jar (363ms)
[info] Done updating.
[error] java.io.FileNotFoundException: 
[error] Could not automatically find the JDK/lib/tools.jar.
[error] You must explicitly set JDK_HOME or JAVA_HOME.
[error]     
[error] 	at org.scaladebugger.SbtJdiTools$.$anonfun$JavaTools$6(SbtJdiTools.scala:47)
[error] 	at scala.Option.getOrElse(Option.scala:121)
[error] 	at org.scaladebugger.SbtJdiTools$.<init>(SbtJdiTools.scala:47)
[error] 	at org.scaladebugger.SbtJdiTools$.<clinit>(SbtJdiTools.scala)
[error] 	at java.base/java.lang.Class.forName0(Native Method)
[error] 	at java.base/java.lang.Class.forName(Class.java:375)
[error] 	at sbt.internal.inc.ModuleUtilities$.getObject(ModuleUtilities.scala:20)
[error] 	at sbt.internal.inc.ModuleUtilities$.getCheckedObject(ModuleUtilities.scala:27)
[error] 	at sbt.internal.inc.ModuleUtilities$.$anonfun$getCheckedObjects$1(ModuleUtilities.scala:31)
[error] 	at scala.collection.immutable.Stream.map(Stream.scala:415)
[error] 	at sbt.internal.inc.ModuleUtilities$.getCheckedObjects(ModuleUtilities.scala:31)
[error] 	at sbt.internal.PluginDiscovery$.loadModules(PluginDiscovery.scala:152)
[error] 	at sbt.internal.PluginDiscovery$.binarySourceModules(PluginDiscovery.scala:144)
[error] 	at sbt.internal.PluginDiscovery$.discover$1(PluginDiscovery.scala:43)
[error] 	at sbt.internal.PluginDiscovery$.discoverAll(PluginDiscovery.scala:53)
[error] 	at sbt.internal.Load$.loadPlugins(Load.scala:1267)
[error] 	at sbt.internal.Load$.loadPluginDefinition(Load.scala:1212)
[error] 	at sbt.internal.Load$.buildPlugins(Load.scala:1191)
[error] 	at sbt.internal.Load$.plugins(Load.scala:1174)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$2(Load.scala:683)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.$anonfun$loadUnit$1(Load.scala:683)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.loadUnit(Load.scala:677)
[error] 	at sbt.internal.Load$.$anonfun$builtinLoader$4(Load.scala:477)
[error] 	at sbt.internal.BuildLoader$.$anonfun$componentLoader$5(BuildLoader.scala:158)
[error] 	at sbt.internal.BuildLoader.apply(BuildLoader.scala:223)
[error] 	at sbt.internal.Load$.loadURI$1(Load.scala:539)
[error] 	at sbt.internal.Load$.loadAll(Load.scala:555)
[error] 	at sbt.internal.Load$.loadURI(Load.scala:485)
[error] 	at sbt.internal.Load$.load(Load.scala:464)
[error] 	at sbt.internal.Load$.$anonfun$apply$1(Load.scala:253)
[error] 	at sbt.internal.Load$.timed(Load.scala:1343)
[error] 	at sbt.internal.Load$.apply(Load.scala:253)
[error] 	at sbt.internal.Load$.defaultLoad(Load.scala:69)
[error] 	at sbt.BuiltinCommands$.liftedTree1$1(Main.scala:716)
[error] 	at sbt.BuiltinCommands$.doLoadProject(Main.scala:716)
[error] 	at sbt.BuiltinCommands$.$anonfun$loadProjectImpl$2(Main.scala:689)
[error] 	at sbt.Command$.$anonfun$applyEffect$4(Command.scala:134)
[error] 	at sbt.Command$.$anonfun$applyEffect$2(Command.scala:130)
[error] 	at sbt.MainLoop$.processCommand(MainLoop.scala:154)
[error] 	at sbt.MainLoop$.$anonfun$next$2(MainLoop.scala:137)
[error] 	at sbt.State$$anon$1.runCmd$1(State.scala:242)
[error] 	at sbt.State$$anon$1.process(State.scala:248)
[error] 	at sbt.MainLoop$.$anonfun$next$1(MainLoop.scala:137)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error] 	at sbt.MainLoop$.next(MainLoop.scala:137)
[error] 	at sbt.MainLoop$.run(MainLoop.scala:130)
[error] 	at sbt.MainLoop$.$anonfun$runWithNewLog$1(MainLoop.scala:108)
[error] 	at sbt.io.Using.apply(Using.scala:22)
[error] 	at sbt.MainLoop$.runWithNewLog(MainLoop.scala:102)
[error] 	at sbt.MainLoop$.runAndClearLast(MainLoop.scala:58)
[error] 	at sbt.MainLoop$.runLoggedLoop(MainLoop.scala:43)
[error] 	at sbt.MainLoop$.runLogged(MainLoop.scala:35)
[error] 	at sbt.StandardMain$.runManaged(Main.scala:113)
[error] 	at sbt.xMain.run(Main.scala:76)
[error] 	at xsbt.boot.Launch$$anonfun$run$1.apply(Launch.scala:109)
[error] 	at xsbt.boot.Launch$.withContextLoader(Launch.scala:128)
[error] 	at xsbt.boot.Launch$.run(Launch.scala:109)
[error] 	at xsbt.boot.Launch$$anonfun$apply$1.apply(Launch.scala:35)
[error] 	at xsbt.boot.Launch$.launch(Launch.scala:117)
[error] 	at xsbt.boot.Launch$.apply(Launch.scala:18)
[error] 	at xsbt.boot.Boot$.runImpl(Boot.scala:56)
[error] 	at xsbt.boot.Boot$.main(Boot.scala:18)
[error] 	at xsbt.boot.Boot.main(Boot.scala)
[error] java.io.FileNotFoundException: 
[error] Could not automatically find the JDK/lib/tools.jar.
[error] You must explicitly set JDK_HOME or JAVA_HOME.
[error]     
[error] Use 'last' for the full log.
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? 
```

this is on MacOS 10.13.6

Java 10, same

**Comment by chipsenkbeil** [comment]
JDK 9+ no longer include jars like tools.jar that contain the relevant classes for the debugger.

According to https://stackoverflow.com/a/35244168/3164172 and other resources I've read, this may be available in the system or boot classloader. Or maybe it isn't necessary at all to do anything, in which this plugin could be a no-op.

**Comment by SethTisue** [comment]
> Or maybe it isn't necessary at all to do anything, in which this plugin could be a no-op.

yeah, I think it could just check the Java version, and if it's at least 9, do nothing.

**Comment by SethTisue** [comment]
PR: https://github.com/chipsenkbeil/sbt-jdi-tools/pull/6

**Comment by SethTisue** [comment]
(once this is fixed, it would be nice to have a 1.1.1 release)

**Comment by chipsenkbeil** [comment]
I'll try to get to that later. Maybe tomorrow. Thanks for the PR.
